README.md

```diff
@@ -8,34 +8,80 @@
 
   <p align="center">
     <br />
-    <a href="https://codemod.com/automations/">Registry</a>
+    <a href="https://codemod.com/registry/">Registry</a>
     ·
     <a href="https://codemod.com/community">Community</a>
     ·
-    <a href="https://docs.codemod.com">Documentation</a>
+    <a href="https://docs.codemod.com">Docs</a>
   </p>
 </p>
 
 ## Codemod.com
 
-At Codemod.com we are building the ecosystem, platform and tooling around code modification bots, aka codemods. The Codemod platform helps you create, distribute, and deploy codemods in codebases of any size.
+At Codemod.com we are building the ecosystem, platform and tooling around code modification bots, aka codemods. Codemod platform helps you create, distribute, and deploy codemods in codebases of any size.
 
 The AI-powered, community-led codemods enable you to automate framework upgrades, large refactoring, and boilerplate programming with unparalleled speed and developer experience.
 
 We are making dream migrations a reality for all developers in the world, one framework at a time.
 
 ## Documentation
 
-For details on how to use the Codemod platform, check out our [documentation](https://go.codemod.com/docs).
+For details on how to use Codemod platform, check out our [documentation](https://go.codemod.com/docs).
+
+## Quickstart
+
+To get started with Codemod platform, you can install Codemod CLI:
```

Comment:
Too many Codemod platform. Just say to get started. Remove the Codemod platform.